notifications_utils/logging.py

```diff
@@ -17,11 +19,63 @@
 logger = logging.getLogger(__name__)
 
 
+def _common_request_extra_log_context():
+    return {
+        "method": request.method,
+        "url": request.url,
+        "endpoint": request.endpoint,
+        # pid and is available on LogRecord by default, as `process` but I don't see
+        # a straightforward way of selectively including it only in certain log messages -
+        # it is designed to be included when the formatter is being configured. This is
+        # why I'm manually grabbing it and putting it in as `extra` here, avoiding the
+        # existing parameter name to prevent LogRecord from complaining
+        "process_": getpid(),
+    }
+
+
 def init_app(app, statsd_client=None, extra_filters: Sequence[logging.Filter] = tuple()):
     app.config.setdefault("NOTIFY_LOG_LEVEL", "INFO")
     app.config.setdefault("NOTIFY_APP_NAME", "none")
     app.config.setdefault("NOTIFY_LOG_PATH", "./log/application.log")
     app.config.setdefault("NOTIFY_RUNTIME_PLATFORM", None)
+    app.config.setdefault(
+        "NOTIFY_REQUEST_LOG_LEVEL",  # noqa
```

Comment:
What's this `# noqa` for?

Reply:
Black was going to put it on one line, which makes the ternary statement look more complicated than it is.

Reply:
Trailing comma fixes that. I think the `# noqa` isn't needed.

At least my black and ruff ignore this with the comment removed because of the trailing comma on the following line.